### ISSUE TITLE
Thumbnail images have no/null alt text

### ISSUE THREAD

**Issue by dotherightthing** [dotherightthing/wpdtrt-gallery#99]
> Links must have discernible text

**Comment by dotherightthing** [comment]
If there's no Alt Text, output the Caption
